docs/src/main/mdoc/testing.md

```diff
@@ -111,6 +111,27 @@ val response: IO[Response[IO]] = service[IO](doesNotMatter).orNotFound.run(
 check[String](response, Status.NotFound, Some("Not found"))
 ```
 
+### Using client
+
+Having HttpApp you can build a client for testing purposes. Following the example above we could define our HttpApp like this:
+
+```scala mdoc:nest
+val httpApp: HttpApp[IO] = service[IO](success).orNotFound
+```
+From this, we can obtain the `Client` instance using `Client.fromHttpApp` and then use it to test our sever/app.
+
+```scala mdoc:nest
+val client = Client.fromHttpApp(httpApp)
+val request: Request[IO] = Request(method = Method.GET, uri = uri"/user/not-used")
+val expectedJson = Json.obj(
```

Comment:
This can also be written like this:
```scala
Json.obj(
  "name" := "johndoe",
  "age" := 42
)
```
which, imo, is a bit easier to read.

Reply:
Sure, no problem!